projects/js-packages/components/package.json

```
@@ -73,7 +73,10 @@
 		"extends @wordpress/browserslist-config"
 	],
 	"exports": {
-		".": "./index.ts",
+		".": {
+			"jetpack:src": "./index.ts",
+			"default": "./build/index.js"
+		},
 		"./tools/jp-redirect": "./tools/jp-redirect/index.ts"
```

Now that I think of it, we should update this entry point too.
```suggestion
		"./tools/jp-redirect": {
			"jetpack:src": "./tools/jp-redirect/index.ts",
			"types": "./build/tools/jp-redirect/index.d.ts",
			"default": "./build/tools/jp-redirect/index.js"
		}
```